.vscode/launch.json

```diff
@@ -11,7 +11,7 @@
             "runtimeExecutable": "yarn",
             "cwd": "${workspaceFolder}/packages/client/hmi-client",
             "runtimeArgs": [
-                "dev"
+                "staging"
```

Comment:
This doesn't impact me, but you should check with your VsCode friends to see if they are okay with this.

Reply:
I am not running via my editor directly so this should not impact me 
I am utilizing the following in `iTerm`
```
./hmiServerDev.sh start staging run
yarn local
```